ADVOCATES.md

```diff
@@ -63,4 +63,5 @@ More information can be found in the [Chef Community Guidelines](https://github.
 * Vacant - Submit a PR to become a Chef Community Advocate
 
 #### Chef_Opscode Reddit ([https://www.reddit.com/r/chef_opscode/](https://www.reddit.com/r/chef_opscode/))
```

Comment:
^ that should probably be renamed to something non-opscode, but r/chef obviously doesn't work and i have no better idea...